biojava-structure/src/main/java/org/biojava/nbio/structure/contact/StructureInterface.java

```diff
@@ -21,10 +21,7 @@
 package org.biojava.nbio.structure.contact;
 
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
I think we decided to avoid * imports.

Reply:
Sorry about that again. I'll revert and set up IntelliJ properly once and for all. FYI this is the way: https://stackoverflow.com/a/3348855/3914327